arch/platform/sky/platform.c

```diff
@@ -85,9 +85,6 @@ force_float_inclusion()
   return __fixsfsi + __floatsisf + __mulsf3 + __subsf3;
 }
 #endif
-/*---------------------------------------------------------------------------*/
-void uip_log(char *msg) { puts(msg); }
```

Comment:
This function is still declared in `uipopt.h`, so should probably be removed from that file too.

Reply:
This was removed in #2966.